app/src/test/contrib/components/ChooseAction/ChooseActionPageTests.tsx

```diff
@@ -25,7 +25,7 @@ describe("Choose Action Page Component", () => {
         expect(typeof rendered.props().onLoad).is.equal('function');
     });
 
-    it("renders component component level", () => {
+    it("renders component on component level, renders title and sub component", () => {
```

Comment:
I'm still not sure I get this. Is it meant to just say "on component level, renders title and sub component"?

Reply:
Now we have this redux/recompose/hoc aproach, we have several render levels on almost each component, especially pages and content components. This render path can be for example, and most common: Connect->branch->Component itself.  Shallow render of enzyme does exactly one level, then if you do .dive() after it, it goes deeper to a next level. In this case component level is next after connect. In many tests its not necessarily first level of component, because sometimes we need to go deeper.